frontend/libs/studio-components/src/components/StudioCodeFragment/StudioCodeFragment.module.css

```diff
@@ -1,8 +1,15 @@
 .code {
+  flex: 1;
   background-color: var(--fds-semantic-surface-neutral-subtle);
   border: 1px solid var(--fds-semantic-border-neutral-subtle);
   border-radius: var(--fds-border_radius-small);
   font-family: 'Courier New', monospace;
   font-size: 0.8em;
   padding: 0 var(--fds-spacing-1);
 }
+
+.customDatamodelBindingOverflow {
+  overflow: auto;
+  height: 2em;
+  overflow-y: hidden;
+}
```

Comment:
[This comment remains relevant](https://github.com/Altinn/altinn-studio/pull/13014#discussion_r1663911625). 

We cannot hardcode the height to 2em here, as it would restrict every StudioCodeFragment using "customDatamodelBindingOverflowStyles" to 2em. Moreover, "customDatamodelBindingOverflowStyles" is set to true by default. Are there instances where this component is used without the need for ellipsis? If so, they may not look like expected within this PR. It would be better to move this logic out of the component and pass className as a prop instead. This approach would also help avoid the complexity of managing component settings through props, which can become hard to maintain over time.

Reply:
I removed it and used className props